<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formUpdBug</class>
 <widget class="QDialog" name="formUpdBug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>SketchFlow Print</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Details Ticket</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Microsoft Yi Baiti</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Titre :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="titleTicket">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>Microsoft Yi Baiti</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Crée par :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="createComboBox">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Microsoft Yi Baiti</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Assigné à :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="usrComboBox">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Microsoft Yi Baiti</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Etat d'avancement :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="etatComboBox">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>ouvert</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>accepté</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>confirmé</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>affecté</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>résolu</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>fermé</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>rejeté</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft Yi Baiti</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Création</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft Yi Baiti</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Fermeture</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft Yi Baiti</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Assignation</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QDateTimeEdit" name="dateCrea">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateEnd">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateAssign">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Microsoft Yi Baiti</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="description"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="save">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Images.qrc">
         <normaloff>:/new/MenuInterface/Ressouces/available_updates-512.png</normaloff>:/new/MenuInterface/Ressouces/available_updates-512.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
